lib/service/user.go

```diff
@@ -165,6 +165,19 @@ func (svc *LndhubService) CalcFeeLimit(destination string, amount int64) int64 {
 	return limit
 }
 
+func (svc *LndhubService) CheckUserBalance(ctx context.Context, amount, userId int64) (result *responses.ErrorResponse, err error) {
+	if svc.Config.MaxAccountBalance > 0 {
+		currentBalance, err := svc.CurrentUserBalance(ctx, userId)
+		if err != nil {
+			return nil, err
+		}
+		if currentBalance+amount > svc.Config.MaxAccountBalance {
```

Comment:
@kiwiidb so far in the other check we only check the settled existing payments and don't include the amount?

what do you think is best here?